apps/icinga/src/main.cpp

```diff
@@ -0,0 +1,91 @@
+#include <Arduino.h>
+#include <otaserver.h>
+#include <kgfx.h>
+#include <HTTPClient.h>
+#include <ArduinoJSON.h>
+
+// CONFIGURATION
+// Please make sure you enabled and configured the new API in your Icinga 2 installation by either setting it up manually
+// or use the CLI command for the automatic setup, which also creates a root-user for the API with a random password (icinga2 api setup)
+// The default port of the Icinga 2 API is 5665, so the URL to go with would be like: https://myicingaserver.com:5665
+// Requires the following API permission(s): status/query
+// Read more about Icinga 2 API Setup: https://icinga.com/docs/icinga-2/latest/doc/12-icinga2-api/
+String icingaAPIURL = "https://hostname:5665";
+String icingaAPIUsername = "username";
+String icingaAPIPassword = "password";
+int refreshTimeInSeconds = 60;
+// -- CONFIGURATION
+
+OTAServer otaserver;
+KGFX ui;
+unsigned long lastTime = 0;
+TFT_eSprite textSpr = ui.createSprite(240, 240);
```

Comment:
With the latest KGFX you can use `ui.createSpriteLarge` which creates an 8-bit color sprite so you don't have to set it again on line 35.